spring-cloud-dataflow-docs/src/main/asciidoc/samples.adoc

```diff
@@ -24,5 +24,3 @@ The samples are part of separate https://github.com/spring-cloud/spring-cloud-da
 
 .Functions
```

Comment:
It might be better to change them tp /htmlsingle/ for consistency.

Reply:
changed everywhere